tutorials/plot_visualize_raw.py

```diff
@@ -28,19 +28,26 @@
 ###############################################################################
 # The channels are color coded by channel type. Generally MEG channels are
 # colored in different shades of blue, whereas EEG channels are black. The
-# channels are also sorted by channel type by default. If you want to use a
-# custom order for the channels, you can use ``order`` parameter of
-# :func:`raw.plot`. The scrollbar on right side of the browser window also
-# tells us that two of the channels are marked as ``bad``. Bad channels are
-# color coded gray. By clicking the lines or channel names on the left, you can
-# mark or unmark a bad channel interactively. You can use +/- keys to adjust
-# the scale (also = works for magnifying the data). Note that the initial
-# scaling factors can be set with parameter ``scalings``. If you don't know the
-# scaling factor for channels, you can automatically set them by passing
-# scalings='auto'. With ``pageup/pagedown`` and ``home/end`` keys you can
-# adjust the amount of data viewed at once. To see all the interactive
-# features, hit ``?`` or click ``help`` in the lower left corner of the
-# browser window.
+# scrollbar on right side of the browser window also tells us that two of the
+# channels are marked as ``bad``. Bad channels are color coded gray. By
+# clicking the lines or channel names on the left, you can mark or unmark a bad
+# channel interactively. You can use +/- keys to adjust the scale (also = works
+# for magnifying the data). Note that the initial scaling factors can be set
+# with parameter ``scalings``. If you don't know the scaling factor for
+# channels, you can automatically set them by passing scalings='auto'. With
+# ``pageup/pagedown`` and ``home/end`` keys you can adjust the amount of data
+# viewed at once. To see all the interactive features, hit ``?`` or click
+# ``help`` in the lower left corner of the browser window.
+#
+# The channels are sorted by channel type by default. You can use the ``order``
+# parameter of :func:`raw.plot <mne.io.Raw.plot>` to group the channels
+# differently. ``order='selection'`` uses the same channel groups as the
+# c-version of mne_browse_raw. It only works for Neuromag data, but
```

Comment:
"as MNE-C's mne_browse_raw" (and I think you can probably add a link to the right part of the C manual, too)